_posts/success_stories/2024-05-20-lavinmq-understanding-code.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "LavinMQ: Understanding code to get the best performance"
+summary: "Building LavinMQ in Crystal was an active choice, as maintaining control over the codebase was a high priority."
+author: beta-ziliani
+image: /assets/blog/2024-05-20-lavinmq-terminal.png
+company: 84codes
+categories: success
+---
+
+**Building LavinMQ in Crystal was an active choice, as maintaining control over the codebase was a high priority. A decade of learning has prioritized fast updates and easy bug fixes. At the same time, choosing Crystal provided the opportunity to enhance application performance.**

```

Comment:
**issue:** I'm not sure highlighting the lead paragraph is a good idea.
We can introduce this, but it should be a concious feature in the visual framework, not an ad-hoc implementation for this single post.

Reply:
I think this summary paragraph makes sense, and it must be highlighted somehow. If any, I would rather keep it as is, and later modify the fwk to include thiis feature.

Reply:
Why "must"? It works perfectly fine without the bold face.
![grafik](https://github.com/crystal-lang/crystal-website/assets/466378/0023f296-edce-4eb0-a418-24c18a802981)

Reply:
there's a history break between the paragraphs. the bold was marking the paragraph as something different than an intro. but it's a nitpick really. I removed the boldface.